src/client/Sidebar/index.tsx

```diff
@@ -63,7 +66,7 @@ export const Sidebar = ({
             <i
               className="bx bxs-file-plus new-btn"
               style={{ color: '#dbdde1' }}
-              onClick={createFile}
+              onClick={() => createFile('New File')}
```

Comment:
Best to move this to a separate definition outside the JSX and use `useCallback`. Or just change `createFile` to default the name.